src/components/StatusButton.tsx

```diff
@@ -79,9 +88,12 @@ const StatusButton = (props: Props) => {
       </Status>
       <ButtonContainer>
         <Button
+          style={{ minWidth: '4.3rem' }}
```

Comment:
This is necessary for look. Otherwise the `Add` and `Upload` have too drastically different widths